test/test_pegen.py

```diff
@@ -367,3 +368,108 @@ def test_cut():
     assert node == [TokenInfo(OP, string='(', start=(1, 0), end=(1, 1), line='(1)'),
                     [TokenInfo(NUMBER, string='1', start=(1, 1), end=(1, 2), line='(1)')],
                     TokenInfo(OP, string=')', start=(1, 2), end=(1, 3), line='(1)')]
+
+
+class TestGrammarVisitor:
+
+    class Visitor(pegen.grammar2.GrammarVisitor):
+        def __init__(self):
+            self.n_nodes = 0
+
+        def visit(self, node):
+            self.n_nodes += 1
+            super().visit(node)
+
+    def test_parse_trivial_grammar(self):
+        # GIVEN
```

Comment:
Is this GIVEN / WHEN / THEN from your corporate style guide? I find it rather verbose.